detox/src/android/expect.test.js

```diff
@@ -125,53 +128,108 @@ describe('expect', () => {
     await expectToThrow(() => e.waitFor('notAnElement').toBeVisible());
   });
 
-  it(`interactions`, async () => {
-    await e.element(e.by.label('Tap Me')).tap();
-    await e.element(e.by.label('Tap Me')).tap({ x: 10, y: 10 });
-    await e.element(e.by.label('Tap Me')).tapAtPoint({x: 100, y: 200});
-    await e.element(e.by.label('Tap Me')).longPress();
-    await e.element(e.by.id('UniqueId819')).multiTap(3);
-    await e.element(e.by.id('UniqueId937')).typeText('passcode');
-    await e.element(e.by.id('UniqueId937')).tapBackspaceKey();
-    await e.element(e.by.id('UniqueId937')).tapReturnKey();
-    await e.element(e.by.id('UniqueId005')).clearText();
-    await e.element(e.by.id('UniqueId005')).replaceText('replaceTo');
-    await e.element(e.by.id('ScrollView161')).scroll(100);
-    await e.element(e.by.id('ScrollView161')).scroll(100, 'down');
-    await e.element(e.by.id('ScrollView161')).scroll(100, 'up');
-    await e.element(e.by.id('ScrollView161')).scroll(100, 'right');
-    await e.element(e.by.id('ScrollView161')).scroll(100, 'left');
-    await e.element(e.by.id('ScrollView161')).scrollTo('bottom');
-    await e.element(e.by.id('ScrollView161')).scrollTo('top');
-    await e.element(e.by.id('ScrollView161')).scrollTo('left');
-    await e.element(e.by.id('ScrollView161')).scrollTo('right');
-    await e.element(e.by.id('ScrollView799')).swipe('down');
-    await e.element(e.by.id('ScrollView799')).swipe('down', 'fast');
-    await e.element(e.by.id('ScrollView799')).swipe('up', 'slow');
-    await e.element(e.by.id('ScrollView799')).swipe('left', 'fast');
-    await e.element(e.by.id('ScrollView799')).swipe('right', 'slow');
-    await e.element(e.by.id('ScrollView799')).swipe('down', 'fast', 0.9);
-    await e.element(e.by.id('ScrollView799')).swipe('up', 'slow', 0.9);
-    await e.element(e.by.id('ScrollView799')).swipe('left', 'fast', 0.9);
-    await e.element(e.by.id('ScrollView799')).swipe('right', 'slow', 0.9);
-    await e.element(e.by.id('ScrollView799')).atIndex(1);
+  describe('element interactions', () => {
```

Comment:
more boyscouting here than really related to this feature, except that I wanted more specific tests related to _this_ specific interaction (screenshot taking)